.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,81 @@
+name: Bug report
+description: Report incorrect behavior in the Modin library
+title: 'BUG: '
+labels: ['bug 🦗', 'Triage 🩹']
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Modin version checks
+      options:
+        - label: >
+            I have checked that this issue has not already been reported.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the latest version of Modin.
+          required: true
+        - label: >
+            I have confirmed this bug exists on the main branch of Modin.
+  - type: textarea
+    id: example
+    attributes:
+      label: Reproducible Example
+      description: >
+        Please follow [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) on how to
+        provide a minimal, copy-pastable example.
+      placeholder: >
+        import modin.pandas as pd
+
+        df = pd.DataFrame(range(5))
+
+        ...
+      render: python
+    validations:
+      required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Issue Description
+      description: >
+        Please provide a description of the issue shown in the reproducible example.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: >
+        Please describe or show a code example of the expected behavior.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Error Logs
+      description: >
+        Please paste the output of any relevant error logs.
+      value: >
+        <details>
+
```

```suggestion
<summary>Stack trace</summary>
```

Do you think this is a good idea?

---

Yeah I think it's ok to add.

---

I don't think it's necessary. I'm wondering if it might confuse folks.